yt_dlp/extractor/roosterteeth.py

```diff
@@ -63,9 +62,9 @@ def _extract_video_info(self, data):
             'display_id': attributes.get('slug'),
             'title': title,
             'description': traverse_obj(attributes, 'description', 'caption'),
-            'series': attributes.get('show_title'),
+            'series': traverse_obj(attributes, 'show_title', 'parent_content_title'),
             'season_number': int_or_none(attributes.get('season_number')),
-            'season_id': attributes.get('season_id'),
+            'season_id': str_or_none(attributes.get('season_id')),
```

Comment:
If it's always a UUID, `str_or_none` is not necessary here. But it probably doesn't hurt, either

Reply:
this field does not exist for bonus features

Reply:
we would still return `None` without `str_or_none` then